core/shared/src/main/scala/zio/logging/package.scala

```diff
@@ -54,6 +54,23 @@ package object logging extends LoggerLayers {
   def loggerName(value: String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] =
     ZIOAspect.annotated(loggerNameAnnotationKey, value)
 
+  /**
+   * Append logger name aspect, by which it is possible to append a logger name to the current logger name.
+   * The new name is appended using a dot (.) as the delimiter.
+   *
+   * annotation key: [[zio.logging.loggerNameAnnotationKey]]
+   */
+  def appendLoggerName(value: String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] =
```

Comment:
just thinking about aspect like

```scala
def setLoggerName(fn: Option[String] => String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] = ...
```
(not sure if it is the best name ...)

and then
```scala
def appendLoggerName(value: String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] =
   setLoggerName(currentLoggerName => currentLoggerName.fold(value)(currentLoggerName => s"$currentLoggerName.$value"))
```

Reply:
make sense, regarding the name, maybe just `loggerName`? It would have different overload than the current one.

Reply:
@somdoron what do you think about that ?

Reply:
not bad idea to use `loggerName`, even if overloads sometimes making problems

Reply:
I pushed a commit with your suggestion, let me know what you think

Reply:
nice,